### PR TITLE
Dev 14218 add cnvrg app cnvrg infra api for east west

### PR DESCRIPTION
**Pull request by MrEsL**

### CODE REVIEW

File: pkg/dbs/tmpl/es/sts.tpl [hunk 1]
```diff
@@ -24,6 +24,9 @@ spec:
         {{$k}}: "{{$v}}"
         {{- end }}
       labels:
+        {{- if and (.Spec.Networking.EastWest.Enabled) (not .Spec.Networking.EastWest.Primary) }}
```
Comment (Dimss) [inline pkg/dbs/tmpl/es/sts.tpl:27]: the `sidecar.istio.io/inject: "true"`, are we planning to manually manage the sidecar injection?

File: pkg/logging/tmpl/elastalert/ingress.tpl [hunk 2]
```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.k8s.io/v1beta1
```
Comment (Dimss) [inline pkg/logging/tmpl/elastalert/ingress.tpl:1]: why do we've an ingress for elastalert? is it something new?

**Review by Dimss — CHANGES_REQUESTED** [review]
@MrEsL

File: pkg/logging/tmpl/elastalert/ingress.tpl (hunk 2 above)
Comment (MrEsL) [reply pkg/logging/tmpl/elastalert/ingress.tpl:1]: I just saw there was vs for elastalert so didnt see a reason for there not to be ingress aswell

File: pkg/dbs/tmpl/es/sts.tpl (hunk 1 above)
Comment (MrEsL) [reply pkg/dbs/tmpl/es/sts.tpl:27]: not sure I understand? the annnotation causes mutation with automatic sidecar

File: api/v1/networking.go [hunk 3]
```diff
@@ -21,6 +21,16 @@ type Istio struct {
 	ExternalIP            []string          `json:"externalIp,omitempty"`
 	LBSourceRanges        []string          `json:"lbSourceRanges,omitempty"`
 	IngressSvcAnnotations map[string]string `json:"ingressSvcAnnotations,omitempty"`
+	EastWest              EastWest          `json:"eastWest,omitempty"`
```
[the remark this answers is not in the document]
Comment (MrEsL) [reply api/v1/networking.go:24]: in cnvrgapp there is not istio struct, so eastwest is under networking. (eastwest is not under ingress?)
in cnvrginfra eastwest is under istio.

**Review by Dimss — COMMENTED** [review]
so far looks good , 
please add tests, 
good job! 
you are the True developer! 
Fucking Golang Cloud Native Developer! 
Ask for 50k :)

**Pull request closed without merging**